src/Raven.Client/Exceptions/Security/InvalidNetworkTopologyException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Raven.Client.Exceptions.Security
+{
+    public class InvalidNetworkTopologyException : SecurityException
```

Comment:
is this a security exception?

Reply:
@ayende Should this be a security exception or something else?

Reply:
No, it should be inheriting from `Exception` directly, instead.

Reply:
Fixed